includes/class-wcpdf-settings-callbacks.php

```diff
@@ -290,38 +288,38 @@ public function radio_button( $args ) {
 	public function multiple_text_input( $args ) {
 		extract( $this->normalize_settings_args( $args ) );
 
-		if (!empty($header)) {
-			echo "<p><strong>{$header}</strong>:</p>";
+		if ( ! empty( $header ) ) {
+			echo wp_kses_post( "<p><strong>{$header}</strong>:</p>" );
 		}
 
-		printf('<p class="%s multiple-text-input">', $id);
+		printf('<p class="%s multiple-text-input">', esc_attr( $id ) );
 		foreach ($fields as $name => $field) {
 			$size = $field['size'];
 			$placeholder = isset( $field['placeholder'] ) ? $field['placeholder'] : '';
 
-			if (isset($field['label_width'])) {
+			if ( isset( $field['label_width'] ) ) {
 				$style = sprintf( 'style="display:inline-block; width:%1$s;"', $field['label_width'] );
 			} else {
 				$style = '';
 			}
 
-			$description = isset( $field['description'] ) ? '<span style="font-style:italic;">'.$field['description'].'</span>' : '';
+			$field_description = ! empty( $field['description'] ) ? '<span style="font-style:italic;">'.$field['description'].'</span>' : '';
 
 			// output field label
-			if (isset($field['label'])) {
-				printf( '<label for="%1$s_%2$s" %3$s>%4$s:</label>', $id, $name, $style, $field['label'] );
+			if ( isset( $field['label'] ) ) {
+				printf( '<label for="%1$s_%2$s" %3$s>%4$s:</label>', esc_attr( $id ), esc_attr( $name ), esc_attr( $style ), esc_attr( $field['label'] ) );
```

Comment:
Here the `$field['label']`is escaped with `esc_attr`, but other labels are with `esc_html`.

Reply:
technically they work the same, but good catch - we should apply them consistently